### PR TITLE
Implemented method for grabbing objects

### PR DESCRIPTION
**Pull request by Abb22TedStr**
A new scene (Grabbing) where you can grab objects.

### CODE REVIEW

**Review by abbolisar — APPROVED** [review]
Merged 🆎

**Review by Abb22TedStr — COMMENTED** [review]
Wait with giving approval until Hampus merges his branch (highlighting objects) with this!

**Comment by Hampus220S** [comment]
I am now done implementing the ability to highlight objects that the user can grab. The program only highlights the closest grab-able object (the one you will grab), but for some unknown reason, the hand doesn't always grab the closest object. That will have to be investigated at some point, but not now :)

How to make objects have the ability to be interact-able, and get an outline, is partly documented in the newly created markdown file called IntegrationInstructions.

**Comment by Hampus220S** [comment]
When this pull request gets closed, also close this issue with the same content:
#7

File: VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineReceiver.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    outline_renderer.material = null;
+
+    // Initially disable the outline object
+    outline_object.SetActive(false);
+  }
+
+  /*
+   * Destory the created outline object
+   */
+  void onDestory()
```
Comment (tramvaj99) [inline VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineReceiver.cs:42]:
```suggestion
  void onDestroy()
```

Should this be onDestroy?

**Review by tramvaj99 — CHANGES_REQUESTED** [review]

File: VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineReceiver.cs (hunk 1 above)
Comment (Hampus220S) [reply VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineReceiver.cs:42]: Maybe... :)

**Review by abbsimmei — APPROVED** [review]
I approve!

**Review by abbsimmei — APPROVED** [review]
I approve again

**Pull request merged**